package-lock.json

```diff
@@ -25,8 +25,8 @@
         "eslint-plugin-jest": "^24.7.0",
         "husky": "^6.0.0",
         "lint-staged": "^11.2.6",
-        "prettier": "^2.5.0",
-        "prettier-plugin-apex": "^1.10.0"
+        "prettier": "2.5.0",
```

Comment:
This file is still showing modified in your PR. Please revert this change.

Reply:
This should be aligned with what's on the main branch.